helm-charts/interoperator/values.yaml

```diff
@@ -28,6 +28,23 @@ broker:
   service:
     type: LoadBalancer
 
+quota_app:
+  port: 9296
+  username: quota
+  password: secret
+  quota:
+    enabled: false
+    oauthDomain: https://myauth-domain.com
+    serviceDomain: https://my-tenant-onboarding.com
+    username: quota_user
+    password: quota_password
+  image:
+    repository: servicefabrikjenkins/service-fabrik-broker
+    tag: 0.6.0
```

Comment:
This tag needs to be changed. Also, since the Jenkins release creattion modifies the tag, please check with @vinaybheri to adjust the automation

Reply:
Another way, will be to remove the image field totally from quota values and use the same value from broker in the quota app deployment.

Reply:
good point. We should just use the same for the broker app, since the image is same.

Reply:
But I think we still have to update tag under broker, right?

Reply:
can you rebase ? It should be updated already

Reply:
https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/helm-charts/interoperator/values.yaml#L26